AndelaOSP/wirebot
# #152107092 Notify Tagged Witnesses

#### What does this PR do?
This PR notifies a `Slack user` via `DM` when they are tagged in an incident report as a witness

#### Description of Task to be completed?
- When an incident is being reported and a Slack user is tagged as a witness, the Slack user is notified via Slack DM that they have been tagged in the incident

#### How should this be manually tested?
Given that you are signed in to a Slack workspace where the bot has been installed:
- Initiate a conversation with the bot
- Indicate that you want to report an incident
- Proceed through the steps of reporting an incident
- Add the witness(es) when requested to do so
- Once you confirm the incident report, the tagged Slack user(s) will receive DM notifications that they've been tagged as witnesses in an incident report

#### Any background context you want to add?
This will notify any users tagged as witnesses to an incident report of the action, from which they can follow up on

#### Screenshots
<img width="917" alt="screen shot 2018-03-27 at 8 40 20 pm" src="https://user-images.githubusercontent.com/6702127/37984538-2d7a3292-31ff-11e8-878c-ce12c9005e17.png">

#### What are the relevant pivotal tracker stories?
[#152107092](https://www.pivotaltracker.com/n/projects/2117172/stories/152107092)
